src/java/io/bazel/rulesscala/worker/Worker.java

```diff
@@ -88,7 +98,8 @@ public void checkPermission(Permission permission) {
             code = e.code;
           } catch (Exception e) {
             System.err.println(e.getMessage());
-            e.printStackTrace();
+            if (e instanceof Interface.WorkerException) {}
```

Comment:
Maybe here instead of empty block you could do `System.err.println(e.getMessage());`?
I think message is printed with `e.printStackTrace();` in an else statement as well. If not then leave this as is.

Reply:
Sure, the `e.printStackTrace` would print the `e.getMessage` so previously there have been exception message printed twice. Let's only print either only exception message or stack trace